relay-server/src/services/store.rs

```diff
@@ -399,6 +425,41 @@ impl StoreService {
         attachment_iterator.chain(event_iterator)
     }
 
+    fn extract_combined_replay_kafka_message(
+        event_id: EventId,
+        replay_items: Vec<&Item>,
+        scoping: Scoping,
+        start_time: Instant,
+        retention: u16,
+    ) -> Option<KafkaMessage> {
+        let mut replay_event_item = None;
+        let mut replay_recording_item = None;
```

Comment:
nit: We could declare these variables in `store_envelope`, so we don't have to transform to and from a vector.